Sources/AppStoreConnectCLI/Services/Operations/ListBetaGroupsOperation.swift

```diff
@@ -38,6 +39,10 @@ struct ListBetaGroupsOperation: APIOperation {
         filters += options.appIds.isEmpty ? [] : [.app(options.appIds)]
         filters += options.names.isEmpty ? [] : [.name(options.names)]
 
+        if let excludeInternal = options.excludeInternal {
+            filters += [.isInternalGroup(["\(excludeInternal)"])]
```

Comment:
They use `String` true or false 🙁 ?

Reply:
yes it lets me input string and "true" or "false" works 😂